charts/azure/azure-disk-loss/experiment.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Detaching the disk attached to azure vm instance
+kind: ChaosExperiment
+metadata:
+  name: azure-disk-loss
+  labels:
+    name: azure-disk-loss
+    app.kubernetes.io/part-of: litmus
+    app.kubernetes.io/component: chaosexperiment
+    app.kubernetes.io/version: latest
+spec:
+  definition:
+    scope: Cluster
+    permissions:
+      - apiGroups: 
+          - "" 
+          - "batch" 
+          - "apps" 
+          - "litmuschaos.io"
+        resources: 
+          - "jobs" 
+          - "pods" 
+          - "pods/log" 
+          - "events" 
+          - "deployments" 
+          - "replicasets" 
+          - "pods/exec" 
+          - "chaosengines" 
+          - "chaosexperiments" 
+          - "chaosresults"
+          - "secrets"
+        verbs: 
+          - "create" 
+          - "list" 
+          - "get" 
+          - "patch" 
+          - "update" 
+          - "delete" 
+          - "deletecollection"
+    image: "litmuschaos/go-runner:latest"
```

Comment:
Just wanted to confirm this, we will be using the `latest` image tag or the `ci` tag? @ispeakc0de @uditgaurav

Reply:
We can keep it uniform right now (with `latest`). Later on, we can discuss and change for all at once if required.